frontend/src/features/frame/components/sidebar/Footer.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+
+import { makeStyles } from '@material-ui/core/styles';
+
+import { handleWikiUrl } from 'src/utils/url';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    padding: 8,
+    borderTop: '1px solid #806300',
+    display: 'flex',
+    flexFlow: 'row wrap',
+    justifyContent: 'space-between',
+  },
+  menuLink: {
```

Comment:
How about adding a `textDecoration: 'underline'` on hover?

The majority of our clickable elements in the UI have a little effect when the mouse is over, and a little visual hint here might help to navigate in the footer.

Reply:
Improved!